server/src/db/migrations/20230504135943_addImageUrlColumn.cjs

```diff
@@ -7,7 +7,7 @@
  */
 exports.up = async (knex) => {
     return knex.schema.alterTable("websites", (table) => {
-        table.string("imageUrl").notNullable().defaultTo(null)
+        table.string("imageUrl").notNullable().defaultTo("")
```

Comment:
there should be no `defaultTo` method if it is setting an empty string - and per Nick's comment to remove it all together

Reply:
Nick fixed this on a the other groups PR so there might be a merge conflict but I want to go with theirs I just had it like this to make sure it seeded properly but when we get both these PRs merged I will make sure we merge the branch that fixes this